### PR TITLE
FPL-104: Content updates for CCD dashboard

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPL-104

### Change description ###
Minor content updates based on UX content review

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseEvent.json [hunk 1]
```diff
@@ -145,7 +145,7 @@
     "LiveFrom": 42748,
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "ID": "attendingHearing",
-    "Name": "Attending the court",
+    "Name": "Attending the hearing",
```
Comment (geek-soft) [inline ccd-definition/CaseEvent.json]: That changes event name in dropdown. Don't we need to make similar change in tab configuration? At the moment tab still displays 'Attending the court' section.

Comment (Jamesnelson117) [reply ccd-definition/CaseEvent.json]: Fair point. Also some minor inconsistencies i.e 'Orders and directions needed' in dropdown but 'Orders and Directions' in tab (casing inconsistency). Going to grab some of Olivia's time to finalise.

Comment (Jamesnelson117) [reply ccd-definition/CaseEvent.json]: Caught up with Oliva. Content in tabs to be consistent with dropdown content. The solicitor table in 'people in the case' should also be nested within the applicant table to reflect 'Applicant' page. Out of scope for this story. 

screen shot 2018-11-05 at 10 55 27

**Comment by sivas23** [comment]
QA - Pass (considering the limitations)

**Pull request merged**